fpdcc/ccfp-asset-dashboard
# calculate political boundaries when an asset is saved + incorporate feature in project overview ui

## Overview

For:
- #126 

### Notes
#### How do we calculate this for production?
All of the projects on the production database already have districts associated with them. New relationships won't be created until they save some assets. Do we delete these existing relationships, or what?

I'm using a new text editor and it appears that I have a plugin installed that removed indentations on lines with no code. Sorry for the extra diff!

## Testing Instructions
- visit review app: https://ccfp-asset-d-feature-pr-awf7kf.herokuapp.com
- login with the `ccfp-asset-dashboard test user` credentials in last pass
- view a project and edit/add a phase. save some assets.
- revisit the project detail page and make sure the districts/zones were updated.
